components/raftstore/src/store/worker/pd.rs

```diff
@@ -659,6 +691,24 @@ where
             Ok(stats) => stats,
         };
 
+        for (region_id, region_peer) in &mut self.region_peers {
+            let read_bytes = region_peer.read_bytes - region_peer.last_store_report_read_bytes;
+            let read_keys = region_peer.read_keys - region_peer.last_store_report_read_keys;
+            // TODO: select hotspot peer by binaray heap in future
+            if read_bytes < hotspot_byte_report_threshold()
+                || read_keys < hotspot_key_report_threshold()
+            {
+                continue;
+            }
+            let mut peer_stat = pdpb::PeerStat::default();
+            peer_stat.set_region_id(*region_id);
+            peer_stat.set_read_bytes(read_bytes);
+            peer_stat.set_read_keys(read_keys);
+            stats.peer_stats.push(peer_stat);
+            region_peer.last_store_report_read_bytes = region_peer.read_bytes;
+            region_peer.last_store_report_read_keys = region_peer.read_keys;
```

Comment:
They also need to be updated although are less than threshold.

Reply:
good catch.